components/src/Tables/Table.js

```diff
@@ -84,6 +115,12 @@ export const CreateRow = styled.tr`
     ${ ActionCell } {
```

Comment:
When is an `ActionCell` not a child of a Row? Why are these styles not defined on the `ActionCell` directly?

Reply:
I believe this is fixed as a result of fixing the previous comment